### DISCUSSION TITLE
Do bulk clothing Modpacks vs. individual modpacks make a difference?

### DISCUSSION THREAD

**Discussion by Breakableoranges** [xivdev/Penumbra#162] [Q&A] [answered]
Had a discussion with some friends about sharing mods. The ideal scenario would be importing all of your clothing and body mods into TexTools, and then exporting a single mod pack with all of those into a single TTMP2 that you can share with friends for their collection of you. Hopefully you also trim it a bit so your personal modpack doesn’t contain unnecessary models for your character, to reduce file size.

A friend said that it’s not a good idea to share entire modpacks of clothing, and definitely not clothing and body combined; that you should do one exported modpack for body mods, and then share a .zip of all of the different individual clothing mod TTMPs. From what I thought, TTMP2s are essentially already zips. Would this truly make a difference in terms of stability when trying to load many different collections at once?

**Comment by Ottermandias** [comment, marked as answer]
You should share your individual mods. You can just zip them up. Why would you go through TexTools at all for that? They're lying around on your filesystem if you use penumbra.

**Reply by Breakableoranges** [reply]
Thanks for the quick reply. So the primary reason for importing them into TexTools first was that, 1) it allowed the “assign body materials” of TexTools to be used before export, fixing texture patching issues that sometimes happen when importing individual mods into Penumbra, especially when trying to use T&F mods with a Bibo+ body. 2) it allowed you to import mods into TexTools, use the item converter, and then have those conversions included when you export them all out of TexTools, so that mods can replace different items than what they were designed for.

I haven’t seemed to find a workaround for those issues that lets me skip TexTools, so please let me know if you know one, because I would love to be able to not use TexTools.

**Comment by Ottermandias** [comment]
You can convert body materials in Penumbra, too. Item Conversion isn't supported. But that's not sharing with a friend, that's editing mods.

You don't need to zip ttmps. You can just zip the mod folders in your penumbra root directory, send it, and your friend can extract the folders in his penumbra root directory.

In any case its a bad idea to put unrelated mod into a single modpack to import them in Penumbra. Why would you do that? You can of course just pull them out separately again, but you could also just keep them separate in the first place, zip both mod folders up and send them.

**Reply by Breakableoranges** [reply]
Thank you. I’ll have to look into the body materials conversion again; I tried to look into it before but had trouble figuring it out.

Zipping the directory files of the mods seems like the best solution, I’ll have to do that. Thanks again for the help, and all the work you’ve put into this.
